code/modules/paperwork/pen.dm

```diff
@@ -110,6 +110,9 @@
 /obj/item/pen/sleepy
 	container_type = OPENCONTAINER
 	origin_tech = "engineering=4;syndicate=2"
+	var/transfer_amount = 50
```

Comment:
why make this a variable if the user cannot vary it? unless I am missing something?

Reply:
Possible admemes. I was thinking along the lines of the changes I made for allowing VV edits with the RSG.